columns/src/Columns.ts

```diff
@@ -0,0 +1,27 @@
+import { ColumnsStyles } from './Columns.styles';
+import { LitElement, html } from 'lit';
+import { property } from 'lit/decorators.js';
+
+/**
+ * Component to render children in a row.
+ */
+export class Columns extends LitElement {
+  static styles = ColumnsStyles;
+
+  @property()
+  gap: string;
```

Comment:
I think it's a css prop now no ?

Reply:
@georges-gomes suggested to use gap as an attribute here https://github.com/divriots/dockit-core/pull/75#discussion_r855759321